content/en/platform/corda/1.6/cenm/_index.md

```diff
@@ -1,16 +1,16 @@
 ---
 date: '2020-01-08T09:59:25Z'
-description: "Documentation for the 1.5 release of Corda Enterprise Network Manager (CENM)"
+description: "Documentation for the 1.6 release of Corda Enterprise Network Manager (CENM)"
 menu:
   versions:
     weight: 290
```

Comment:
I think this needs to be updated to correct the order on the "Corda" drop-down menu.

Reply:
Updated